### PR TITLE
Whirlwind skill

### PR DESCRIPTION
**Pull request by NegativeScript**

### CODE REVIEW

File: tests/furusiya/model/skills/test_whirlwind.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+def bushslime():
+    yield _mock_factory(x=7, y=8)
+
+
+@pytest.fixture
+def _fighter():
+    yield Mock()
+
+
+@pytest.fixture
```
Comment (nightblade9) [inline tests/furusiya/model/skills/test_whirlwind.py]: You don't need test fixtures for everything. You can just use local variables.

File: tests/furusiya/model/skills/test_whirlwind.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+def tigerslash():
+    yield _mock_factory(x=2, y=2)
+
+
+@pytest.fixture
+def whirlwind(player, player_fighter, bushslime, tigerslash, _fighter):
+    FighterSystem.set_fighter(player, player_fighter)
+    FighterSystem.set_fighter(bushslime, _fighter)
+    FighterSystem.set_fighter(tigerslash, _fighter)
```
Comment (nightblade9) [inline tests/furusiya/model/skills/test_whirlwind.py]: Please add a second enemy in range, eg. at `(2, 1)` and assert that both get hit.

File: tests/furusiya/model/skills/test_whirlwind.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+
+@pytest.fixture
+def _fighter():
+    yield Mock()
+
+
+@pytest.fixture
+def tigerslash():
+    yield _mock_factory(x=2, y=2)
```
Comment (nightblade9) [inline tests/furusiya/model/skills/test_whirlwind.py]: Can you use relative coordinates here? That would make it clearer.

Comment (NegativeScript) [reply tests/furusiya/model/skills/test_whirlwind.py]: Relative coordinates? Relative to what, exactly?

**Comment by nightblade9** [comment]
LGTM. Relative coordinates (monsters relative to the player) would be AWESOME, but this is fine.

**Pull request merged**